jmix-bom/bom.gradle

```diff
@@ -391,6 +391,11 @@ dependencies {
 
         api 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.17.3'
 
+        api 'gr.netmechanics.jmix:azurefs:1.3.1'
```

Comment:
Please move it right under the `// community add-ons` comment at the top of the dependencies configuration.

Reply:
ready